src/sentry/api/serializers/models/user_identity_config.py

```diff
@@ -110,6 +113,29 @@ def get_model_type_for_category(self) -> type:
 
 @register(UserIdentityConfig)
 class UserIdentityConfigSerializer(Serializer):
+    def get_attrs(
+        self, item_list: Sequence[UserIdentityConfig], user: User
+    ) -> MutableMapping[UserIdentityConfig, MutableMapping[str, Any]]:
+        org_ids = [i.organization.id for i in item_list if i.organization]
+        organizations = {
+            org.id: dataclasses.asdict(org)
```

Comment:
as clever as this is, I worry about overly relying on the serialization semantics of our objects.  I'll look at seeing if using an explicit `organization_service.serialize_organizations` makes more sense.

Reply:
Ahhhh disregard, this is just the get attrs, which ends up being split up for the... yeah ok I see how this works.  I forget serializers work in this weird way.